ee-feature-pack/galleon-common/src/main/resources/layers/standalone/web-passivation/layer-spec.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" ?>
-<layer-spec xmlns="urn:jboss:galleon:layer-spec:1.0" name="distributable-web">
```

Comment:
FYI - this should use the distributable-web-local feature-group, not distributable-web.
Currently, that feature-group is buggy, as it specifies parameters that are not valid.
See https://github.com/wildfly/wildfly/pull/13818/commits/4aaa6d224e3705f80fcfb73bb4811e9f97b49ca1#diff-fae6397e0df9fed6b193c54885a6b257568d2a361e472a17c55566afd18aa256L18